controllers/front/ProductController.php

```diff
@@ -1117,7 +1119,7 @@ public function getTemplateVarProduct()
         $product['minimal_quantity'] = $this->getProductMinimalQuantity($product);
         $product['quantity_wanted'] = $this->getRequiredQuantity($product);
         $product['extraContent'] = $extraContentFinder->addParams(['product' => $this->product])->present();
-
+        $product['ecotax_tax_inc'] = $this->product->getEcotax(null, true, true);
```

Comment:
Is it related to PR? If not, maybe shouldn't add it even if it fixes something? Better to raise separate issue, because the PR has very clear scope and is already huge.

Reply:
You are right, this is just "saw it, removed it". I can open another PR to clean this up.

Reply:
Moved into separate PR https://github.com/PrestaShop/PrestaShop/pull/29462